enterprise/utils.py

```diff
@@ -749,35 +749,63 @@ def get_course_run_start(course_run, default=None):
     return parse_datetime_handle_invalid(course_run.get('start')) or default
 
 
-def get_current_course_run(course):
+def get_closest_course_run(course_runs):
     """
-    Return the current course run.
+    Return course run with start date closest to now.
+    """
+    if len(course_runs) == 1:
```

Comment:
Question about this check -- it seems to assume that a learner is only enrolled in one run of a course, and if so it will short-circuit and return that run back to the caller.  However, the scenario that was presented which triggered this PR was that learners were enrolled in multiple runs of a course.  In this case, will the short-circuit be bypassed because there are two active runs for the course?  Which could potentially lead to a third run being selected as "current" at some point in the future, and then possibly a fourth, etc.?

Reply:
So basically there are two use cases when a user request for course

1. A user is coming for the first time for that course and a user has no clue for which course run to go in that case user will go to the closest course run. It was previously implemented and working the same.

2. User is already enrolled in one or more course runs. In that case, user will always land on the already enrolled course run, if user is enrolled in one course run then the user will land to that course run this short circuiting will be responsible for that. But if the user is enrolled more than 1 course run then we will find the closest one from those enrolled course runs. **The egde case was user will never be go to the next course run automtically.** User will be enrolled to the next course run manually by admin/enterprise admin.
FYI, @ziafazal @georgebabey